### PR TITLE
Document Alt texture items

### LINKED ISSUE #643: Finish implementing alternate item textures
From @walksanatora and @Talia-12 on Discord:

image
image

Although this is an enhancement, I'm prioritizing it higher than other new stuff because I feel like we should try to avoid releasing half-finished features if possible.

### PR DESCRIPTION
**Pull request by navarchus**
Fixes #643

### CODE REVIEW

File: Common/src/main/resources/assets/hexcasting/lang/en_us.flatten.json5 [hunk 1]
```diff
@@ -1868,7 +1869,9 @@
       },
       
       colorize: "I must be holding a $(l:items/pigments)$(item)Pigment/$ in my other hand to cast this spell. When I do, it will consume the dye and permanently change my mind's coloration (at least, until I cast the spell again). Costs about one $(l:items/amethyst)$(item)Amethyst Dust/$.",
-      
+
+      cycle_variant: "Certain items I create seem oddly receptive to the influence of _media. By holding a $(l:items/hexcasting)$(item)Cypher/$, $(l:items/hexcasting)$(item)Trinket/$, $(l:items/hexcasting)$(item)Artifact/$ or $(l:items/spellbook)$(item)Spellbook/$ in my other hand I can change the appearance of the item. Costs about one $(l:items/amethyst)$(item)Amethyst Dust/$.",
```
Comment (object-Object) [inline Common/src/main/resources/assets/hexcasting/lang/en_us.flatten.json5]:
```suggestion
      cycle_variant: "Certain items I create seem oddly receptive to the influence of _media. By holding a $(l:items/hexcasting)$(item)Cypher/$, $(l:items/hexcasting)$(item)Trinket/$, $(l:items/hexcasting)$(item)Artifact/$, $(l:items/focus)$(item)Focus/$, or $(l:items/spellbook)$(item)Spellbook/$ in my other hand, I can use this spell to change the appearance of the item. Costs about one $(l:items/amethyst)$(item)Amethyst Dust/$.",
```

**Comment by gamma-delta** [comment]
LGTM

**Comment by gamma-delta** [comment]
Oh shoot what.

**Comment by gamma-delta** [comment]
Why does ctrl+shift+enter close it automatically I hate that. I have too much gitlab muscle memory

**Pull request merged**